crypto/rand_test.go

```diff
@@ -0,0 +1,18 @@
+package crypto
+
+import (
+	"testing"
+)
+
+// TestRandIntnPanics tests that RandIntn panics if n <= 0. The crypto/rand
+// package guarantees that rand.Int will panic if n <= 0, but other random
+// packages might not.
```

Comment:
No need to write a justification comment (the second sentence). If something is supposed to have a certain behavior, that's all the reason you need to test that the behavior actually exists.